### PR TITLE
IAP-5638: update upgrade from prior to may22 docker section

### PR DESCRIPTION
**Pull request by kvedenskii-axway**
http://pr-2474.axway-open-docs.opendocs-builder.pcloud.axway.int/docs/apim_installation/apiportal_docker/docker_portal_upgrade/#upgrade-from-versions-before-may-2022-to-latest-release

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | 0bbfdfb |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-2474.axway-open-docs.opendocs-builder.pcloud.axway.int |

**Review by dginevaAxway — APPROVED** [review]
Looks like what we discussed, more simplified. @andreamussap  we move some of the specific steps in to May 22 and August 22 readme filed associated to Docker images. Not in release notes.

File: content/en/docs/apim_installation/apiportal_docker/docker_portal_upgrade.md [hunk 1]
```diff
@@ -34,42 +34,20 @@ There is no support for downgrading an API Portal container deployment. Running
...
-7. Click **Components > Joomla! Update > Upload & Update**, then apply *Joomla 4* by uploading the relevant file from third party package.
-8. Wait for the upgrade process to finish and log in to JAI again.
-9. Click **System > Install > Extensions > Upload Package File**, then apply the *T3 System* plugin by uploading the relevant file from third party package.
-10. Stop the `feb22` container and start a new one from the latest image with the old parameters, but without the mounted `apiportal.ini` file.
+1. If your API Portal version is lower than [February 2022](/docs/apim_relnotes/20220228_apip_relnotes/), you must first upgrade to February 2022 as described in the [General upgrade](#general-upgrade) section.
+2. It is important that you use plain MySQL connection. So, you ensure the `MYSQL_SSL_ON` environment variable is set to `0`, and `MYSQL_USER` and `MYSQL_PASSWORD` environment variables contain credentials for plain connection. Also, you might need to [create a MySQL user account without TLS authentication](/docs/apim_installation/apiportal_install/install_software_configure_database/#configure-a-user-account-without-authentication).
```
Comment (andreamussap) [inline content/en/docs/apim_installation/apiportal_docker/docker_portal_upgrade.md:42]: (It is important that you use plain MySQL connection) --> Is this a step, an action? It's important to use plain connection where/when?

File: content/en/docs/apim_installation/apiportal_docker/docker_portal_upgrade.md [hunk 2]
```diff
@@ -34,42 +34,20 @@ There is no support for downgrading an API Portal container deployment. Running
 
 ## Upgrade from versions before May 2022 to latest release
 
-API Portal versions after [February 2022](/docs/apim_relnotes/20220228_apip_relnotes/) (7.7.20220228) is integrated with *Joomla 4*, which results in some backward incompatible changes. Attempting to upgrade directly from versions prior to [May 2022](/docs/apim_relnotes/20220530_apip_relnotes/) to latest will break database integrity. To upgrade from May 2022 to API Portal lates 2022 release, follow [General upgrade](#general-upgrade) section above in this page.
+API Portal versions after [February 2022](/docs/apim_relnotes/20220228_apip_relnotes/) (7.7.20220228) is integrated with *Joomla 4*, which results in some backward incompatible changes. Attempting to upgrade directly from versions prior to [May 2022](/docs/apim_relnotes/20220530_apip_relnotes/) to latest will break database integrity. To upgrade from versions higher than February 2022 to API Portal latest release, follow [General upgrade](#general-upgrade) section.
```
Comment (andreamussap) [inline content/en/docs/apim_installation/apiportal_docker/docker_portal_upgrade.md:37]: Please fix the following: 

"API Portal **versions** after February 2022... **is** integrated with..." --> "**are**" integrated with....

Comment (kvedenskii-axway) [reply content/en/docs/apim_installation/apiportal_docker/docker_portal_upgrade.md]: Fixed

File: content/en/docs/apim_installation/apiportal_docker/docker_portal_upgrade.md [hunk 3]
```diff
@@ -34,42 +34,20 @@ There is no support for downgrading an API Portal container deployment. Running
...
-8. Wait for the upgrade process to finish and log in to JAI again.
-9. Click **System > Install > Extensions > Upload Package File**, then apply the *T3 System* plugin by uploading the relevant file from third party package.
-10. Stop the `feb22` container and start a new one from the latest image with the old parameters, but without the mounted `apiportal.ini` file.
+1. If your API Portal version is lower than [February 2022](/docs/apim_relnotes/20220228_apip_relnotes/), you must first upgrade to February 2022 as described in the [General upgrade](#general-upgrade) section.
+2. It is important that you use plain MySQL connection. So, you ensure the `MYSQL_SSL_ON` environment variable is set to `0`, and `MYSQL_USER` and `MYSQL_PASSWORD` environment variables contain credentials for plain connection. Also, you might need to [create a MySQL user account without TLS authentication](/docs/apim_installation/apiportal_install/install_software_configure_database/#configure-a-user-account-without-authentication).
+3. Stop the `feb22` container and start a new one from the latest image with the old parameters.
```
[the remark this answers is not in the document]
Comment (kvedenskii-axway) [reply content/en/docs/apim_installation/apiportal_docker/docker_portal_upgrade.md]: Fixed

File: content/en/docs/apim_installation/apiportal_docker/docker_portal_upgrade.md (hunk 1 above)
Comment (kvedenskii-axway) [reply content/en/docs/apim_installation/apiportal_docker/docker_portal_upgrade.md]: Fixed. As for "might", I changed it to "see ... for". There is no need to force creating a new non-TLS database account if there is an existing one

**Pull request merged**